src/lib.rs

```diff
@@ -537,12 +537,20 @@ pub fn get_if_addrs() -> io::Result<Vec<Interface>> {
 #[cfg(test)]
 mod test {
     use super::get_if_addrs;
+    use ip::IpAddr;
+    use std::net::Ipv4Addr;
 
     #[test]
     fn test_get_if_addrs() {
         let ifaces = get_if_addrs().unwrap();
```

Comment:
This should be perhaps 2 tests, on for local_addr == 1 and another that compares what the OS believes is ifaddrs with what we believe, i.e. compare parsed output of ipconfig with our output or use /proc filesystem to get the interface addresses